Sources/Fullscreen/Explore/ViewModels/BrazeBannerViewModel.swift

```diff
@@ -0,0 +1,9 @@
+import FinniversKit
+
+class BrazeBannerViewModel: UniqueHashableItem {
+    let brazePromo: BrazePromotionView?
```

Comment:
Can we make `brazePromo` required? I guess there won't be a case where we provide a `.brazePromo` with no promo inside? 🤔

Reply:
Can do :)